sass/themes/frost/components/_modal.scss

```diff
@@ -0,0 +1,37 @@
+.modal__overlay {
+    position: fixed; /* Stay in place */
+    z-index: 1; /* Sit on top */
+    left: 0;
+    top: 0;
+    right: 0;
+    bottom: 0;
+    width: 100%; /* Full width */
+    height: 100%; /* Full height */
+    overflow: auto; /* Enable scroll if needed */
+    background-color: rgb(0,0,0); /* Fallback color */
```

Comment:
OK, makes sense as this is in frost only components @lebaz20. Merging this version now.